docs/examples/text_classification.md

```diff
@@ -1,31 +1,163 @@
 This example shows how to build a text classifier with Ludwig.
-It can be performed using the [Reuters-21578](http://boston.lti.cs.cmu.edu/classes/95-865-K/HW/HW2/reuters-allcats-6.zip) dataset, in particular the version available on [CMU's Text Analytics course website](http://boston.lti.cs.cmu.edu/classes/95-865-K/HW/HW2/).
-Other datasets available on the same webpage, like [OHSUMED](http://boston.lti.cs.cmu.edu/classes/95-865-K/HW/HW2/ohsumed-allcats-6.zip), is a well-known medical abstracts dataset, and [Epinions.com](http://boston.lti.cs.cmu.edu/classes/95-865-K/HW/HW2/epinions.zip), a dataset of product reviews, can be used too as the name of the columns is the same.
 
-| text                                                                                            | class       |
-| ----------------------------------------------------------------------------------------------- | ----------- |
-| Toronto  Feb 26 - Standard Trustco said it expects earnings in 1987 to increase at least 15...  | earnings    |
-| New York  Feb 26 - American Express Co remained silent on market rumors...                      | acquisition |
-| BANGKOK  March 25 - Vietnam will resettle 300000 people on state farms known as new economic... | coffee      |
+These interactive notebooks follow the steps of this example:
 
+- Ludwig CLI: [![Text Classification with Ludwig CLI](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ludwig-ai/ludwig-docs/blob/daniel/text_classification/docs/examples/text_classification/Text_Classification_with_Ludwig_CLI.ipynb)
+- Ludwig Python API: [![Text Classification with Ludwig Python API](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/ludwig-ai/ludwig-docs/blob/daniel/text_classification/docs/examples/text_classification/Text_Classification_with_Ludwig_Python_API.ipynb)
+
+We'll be using AG's news topic classification dataset, a common benchmark dataset for text classification. This dataset
+is a subset of the full AG news dataset, constructed by choosing the four largest classes from the original corpus. Each
+class contains 30,000 training samples and 1,900 testing samples. The total number of training samples is 120,000 with
+7,600 total testing samples.
+
+This dataset contains four columns:
+
+| column      | description                                                                    |
+| ----------- | ------------------------------------------------------------------------------ |
+| class_index | An integer from 1 to 4: "world", "sports", "business", "sci_tech" respectively |
```

Comment:
we have changed this in the preproc right? there's not integer anymore

Reply:
I added column `class` in preprocessing, but I didn't replace class_index.